addons/project-info/userscript.js

```diff
@@ -23,8 +23,16 @@ export default async function ({ addon, console, msg }) {
         projectInfo = { spriteCount: "?", scriptCount: "?" };
       }
     }
-    container.appendChild(msg(projectInfo.spriteCount == 1 ? "sprite" : "sprites", { num: projectInfo.spriteCount }));
+    container.appendChild(
+      document.createTextNode(
+        msg(projectInfo.spriteCount == 1 ? "sprite" : "sprites", { num: projectInfo.spriteCount })
```

Comment:
How this is done will change after the next l10n PR is merged, so I'm keeping this PR pending for now.

Reply:
What needs be changed here?